azure/Guidelines.md

```diff
@@ -846,109 +848,189 @@ https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#perfo
+```
 
-<a href="#lro-status-monitor-get-returns-200" name="lro-status-monitor-get-returns-200">:white_check_mark:</a> **DO** support the GET method on the status monitor endpoint that returns a `200-OK` response with the current state of the status monitor.
+- This operation *explicitly* creates a status monitor resource as passed in the request body.
 
-<a href="#lro-status-monitor-accepts-any-api-version" name="lro-status-monitor-accepts-any-api-version">:ballot_box_with_check:</a> **YOU SHOULD** allow any valid value of the `api-version` query parameter to be used in the get operation on the status monitor.
+  - The request's `kind` property must be set by the client to some value pre-defined by the service; for example `batchDocumentTranslate`.
 
-Note: Clients may replace the value of `api-version` in the `Operation-Location` URI with a value appropriate for their application.
+- The response body is the status monitor resource whose schema must be idential to the request's body.
 
-<a href="#lro-status-monitor-structure" name="lro-status-monitor-structure">:white_check_mark:</a> **DO** return a status monitor in the response body that conforms with the following structure:
+- The `operation-id` URL segment (not header) is *mandatory* forcing the client to specify the status monitor's resource ID and is also used for retries/idempotency. 
 
-**OperationStatus** : Object
+## The Status Monitor Resource
+All patterns that initiate a LRO either implicitly or explicitly create a [Status Monitor resource](https://datatracker.ietf.org/doc/html/rfc7231#section-6.3.3) in the service's `operations` collection. A Status Monitor resource must look like this:
 
 Property | Type        | Required | Description
 -------- | ----------- | :------: | -----------
 `id`     | string      | true     | The unique id of the operation
-`status` | string      | true     | enum that includes values "NotStarted", "Running", "Succeeded", "Failed", and "Canceled"
-`error`  | ErrorDetail |          | Error object that describes the error when status is "Failed"
-`result` | object      |          | Only for POST action-type LRO, the results of the operation when completed successfully
-additional<br/>properties | |     | Additional named or dynamic properties of the operation
+`kind`   | string enum | true     | The kind of operation
+`status` | string enum | true     | The operation's current status: "NotStarted", "Running", "Succeeded", "Failed", and "Canceled"
+`error`  | ErrorDetail |          | If `status`=="Failed", contains reason for failure
+`result` | object      |          | if Action LRO (POST or PUT) && `status`=="Succeeded", contains success result
+additional<br/>properties | | | Additional named or dynamic properties of the operation
+
+- Because services can support different kinds of operations, status monitor resources in this collection *must be* polymorphic; the `kind` property indicates the kind of long-running operation.
```

Comment:
A single bullet below the table? Seems odd. I have a more general comment about all these bullets in my closing remark for the PR, though...